waspc/e2e-test/ShellCommands.hs

```diff
@@ -51,6 +54,9 @@ runShellCommandBuilder shellCommandBuilder context =
 combineShellCommands :: [ShellCommand] -> ShellCommand
 combineShellCommands = intercalate " && "
 
+pipeShellCommands :: [ShellCommand] -> ShellCommand
```

Comment:
Suggestion: define new operator that pipes two commands and use that, resulting code will be more readable I think. Ideas for name: `.|.` or `$|`.